src/web/command_control/client/components/mission/RunItem.tsx

```diff
@@ -66,10 +66,15 @@ export default class RunItem extends React.Component<RunItemProps, RunItemState>
         this.props.setOpenRunPanels(this.props.openRunPanels)
     }
 
-    handleDuplicateRunClick(evt: React.MouseEvent<HTMLButtonElement>) {
+    /**
+     * On user click the current run will be copied and 
+     * a new run will be created from this copied run
+     * so that we can have multiple bots running the same run.
```

Comment:
maybe "multiple bots with the same run"? running the same run sounds a little off but that could just be me